maxim/logger/langchain/tracer.py

```
@@ -352,7 +381,7 @@ def on_llm_end(self, response: LLMResult, **kwargs: Any) -> Any:
         try:
             scribe().debug("[MaximSDK][Langchain] on_llm_end called")
             run_id = kwargs.get("run_id", None)
-            parent_run_id = kwargs.get("parent_run_id", None)       
+            parent_run_id = kwargs.get("parent_run_id", None)
```

Ruff (0.11.9)

Use `kwargs.get("parent_run_id")` instead of `kwargs.get("parent_run_id", None)`

Replace `kwargs.get("parent_run_id", None)` with `kwargs.get("parent_run_id")`

(SIM910)